src/repmat.c

```diff
@@ -41,12 +42,11 @@ int main_repmat(int argc, char* argv[])
 	int rep = atoi(argv[2]);
 
 	assert(dim < DIMS);
-	assert(rep >= 0);
-	assert(1 == in_dims[dim]);
```

Comment:
I don't see how this could work. If in_dims > 1, the strides would not be zero, so it would not replicate this input....